src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
Cool, so I think using our own OG group in the annotation should stay in that case.